### ISSUE TITLE
Old version.

### ISSUE THREAD

**Issue by vchunkyzv** [MarkusMcNugen/docker-qBittorrentvpn#170]
I'm currently running v4.3.8 Web while there is a v4.6.7 desktop version out. Will this be updated soon?

**Comment by mastakebob** [comment]
+1.  I'm looking forward to the "block certain extensions" functionality so I can auto-block those pesky .lnk malwares...

**Comment by vchunkyzv** [comment]
That’s EXACTLY what led me here and to post that. Lol.

**Comment by mastakebob** [comment]
Yea, looking at the main page, this developer hasn't updated in 3 years. Might be abandoned. 

I changed my docker image over to one maintained by "dyonr". Very easy, just changed 'markusmcnuggen' to 'dyonr' in my docker compose. All other configs stayed the same. Uses the latest qbittorrent version. 

FYI...

**Comment by vchunkyzv** [comment]
Yeah I saw that it had been a while and decided to post in case the developer decides to hop back on. I’m gonna look into the one you said but I have a synology server that is running this. I used this guide to get it all set up: https://www.blackvoid.club/qbittorrent-via-vpn-docker-container-running-on-synology-nas/

and thought it would be easier if the image gets updated but don’t know how easy it would be changing to the one you said. Hopefully it’s easier than I think. Lol

**Comment by vchunkyzv** [comment]
I just set it all up and it's up and running with the torrents I was seeding on the client. I was afraid I was going to lose what I had in there. I added *.lnk

**Comment by vchunkyzv** [comment]
BTW, and in hopes of helping anyone else, I tried the dyonr image on a synology on DSM-7 but there seems to be an issue on DSM 7 and even QNAP about the iptables. https://github.com/Trigus42/alpine-qbittorrentvpn/issues/52. I had to revert to using the original container as it seems right now that we cannot update past that on DSM 7.

**Comment by dmudgeau** [comment]
I also tried the dyonr version but it didn't work got issues with dev/tun not existing and then another issue this is the only one that worked
